Week 4/Unit 08/Assignment 2/lab/src/main/java/fpt/lab/config/ContactDataSourceConfig.java

```diff
@@ -0,0 +1,30 @@
+package fpt.lab.config;
+
+import javax.sql.DataSource;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+
+@Configuration
+public class ContactDataSourceConfig {
+    
+    @Bean(name="contactDataSource")
+    public DataSource contactDataSource() {
+        HikariConfig config = new HikariConfig();
+        config.setDriverClassName("com.mysql.cj.jdbc.Driver");
```

Comment:
Pls put database info in application prop file and get by @Value.

Reply:
Thank you for the correction. I already update it on the last commit.